### ISSUE TITLE
Add accessibility information to the send button

### ISSUE THREAD

**Issue by BasThomas** [GitHawkApp/MessageViewController#56]
<img width="576" alt="screen shot 2018-04-10 at 11 44 13" src="https://user-images.githubusercontent.com/4190298/38549534-8169a170-3cb4-11e8-971d-62276376c59e.png">

**Comment by gingerbeardman** [comment]
I did not know that you could inspect the simulator like this! Very cool.

**Comment by rnystrom** [comment]
@BasThomas should we just be assigning the AX properties here?

https://github.com/GitHawkApp/MessageViewController/blob/master/MessageViewController/MessageView.swift#L132-L142

Or like what info are we missing?

**Comment by BasThomas** [comment]
Yeah, the only tricky thing is that at that point they would always be tied to the information passed into the function — if someone would want a different `accessibilityLabel` than the `title`, they would not be able to do so.

So we could add an optional parameter that would override the accessibilityLabel, just not sure if that's scalable enough / overkill.
